### ISSUE TITLE
What about `className` attribute be concatenated at the end instead of beginning?

### ISSUE THREAD

**Issue by tiagostutz** [MathiasGilson/Tailwind-Styled-Component#24]
Hi there! 👋🏻 

Considerate the following simple scenario:
```javascript
// H1.js
import tw from "tailwind-styled-components";

export const H1 = tw.h1`
    text-3xl 
    text-blue-700 
    font-bold
`;

```

```javascript
// Home.js
...
  <H1 className="mt-4 pb-6">Hallo!</H1>
...
```

To me the final `className` attribute would be **"text-3xl text-blue-700 font-bold mt-4 pb-6"**. But instead, the outer most `className` attribute is being placed at the beginning of the className yielding to **"mt-4 pb-6 text-3xl text-blue-700 font-bold"**. But, IMHO, in the described scenario the parameter passed to the component should be concatenated at the end of the `className` attribute allowing, among other things, overwrite the classes defined at the component code.
I checked the code and it looks like that it concatenates the `template` classes with the `inheritedClasses`. But I think that the "newClasses" in this scenario would be the ones being received as parameter by the use of the component.

What do you think? I'm available to create the PR if this makes sense.

**Comment by AlexMason** [comment]
This was my bad when I fixed a previous issue,  line 26 just needs the parameters swapped and it will be fixed.  Create the PR and hopefully it will get accepted in a reasonable time.

**Comment by tiagostutz** [comment]
Hi @AlexMason ! 

Nice. I'll work on this PR and will hopefully open the PR asap.

Also, I was introduced to this project: https://github.com/ben-rogerson/twin.macro

Do you know this project also?

**Comment by MathiasGilson** [comment]
Hello @tiagostutz, 

This should be fixed in the 2.1.0 thanks to @Dudeonyx PR!
